pwiz_tools/Skyline/Model/Results/ChromHeaderInfo.cs

```diff
@@ -1638,6 +1642,29 @@ public bool UsedMs2Centroids
             get { return UsedMs2CentroidsFlags(Flags); }
         }
 
+        public bool IsSrm

```

Comment:
I prefer having a single way of doing things to avoid confusion. Can you either make the other getters that call private static functions do their flag manipulation in-line or give this a parallel private static function? I am fine with either way. Maybe this way is better since it keeps others from wondering why the extra functions were defined, as I wonder, unless it is simply to be ready in case they need to be made public like IsSingleMatchMzFlags() and IonMobilityUnitsFromFlags().